daterangepicker.js

```diff
@@ -1289,6 +1306,23 @@
                     }
 
                 });
+            } else if(this.flag || this.container.find('input[name=daterangepicker_start]').is(":focus")) {
+                this.container.find('.calendar tbody td').each(function(index, el) {
+
+                    //skip week numbers, only look at dates
+                    if ($(el).hasClass('week')) return;
+
+                    var title = $(el).attr('data-title');
+                    var row = title.substr(1, 1);
+                    var col = title.substr(3, 1);
+                    var cal = $(el).parents('.calendar');
+                    var dt = cal.hasClass('left') ? leftCalendar.calendar[row][col] : rightCalendar.calendar[row][col];
```

Comment:
In fact you could remove line number 1315-1319 and it would have no effect on the code.